.travis.yml

```diff
@@ -0,0 +1,57 @@
+  - secure: d+4aM/IGqszna5Yv7M+6wWmdzafSkPH5qJtvd5Jac6QUxMhDIzIN+br+Sf2PNougZso7EsaFHlZQKatSmCM+DsVy2Ke5DB82icTsMyBzano8GF+eYm9BJUhgOk933ygsDKRFvfaiwAPjGgyFDwLAc4CPc/Lly/mqs4wyyekFgTWvIF+AG9GaigCkQB14Ud70BJ0PBCGh82lqKSY32r7ti/G5uNrD5DFIOpUmoq9NWeX7tAvWi6iWWCdndmAnT7aRpb7oprgMTm4Jc7CguJvz2pKh6a2ncY29nuvr0DNEfXRGiGKAZAck852ad/pwSa04Uwi1j5arFAZuRgtldhksFQPWZM4s7SP9e6LpSWU4+QyuXY5R1Rs5KtdREd3J27z2RmFm0m5vMkUTGMMHvw+qOrITVovWKme+ptOKytpIVgoxmzA/Y2/aS0JxN78nuLfX5v24+yDQ1aIxtjTzQufkGyFPMqwDPV0ZO5/Ez0FP1lPWd5cRrL77ako1sTVje4Up08VwcmJueJH2Xv7YfmvYR0w/VMymEliKpDsTJ6A7/o+Y2gyi6mwTjg7q/6aXTBoxVH/3LFABJeBQWqhIaDTdk5xjutEGUKyUMoHjW6fqTvoErLRfwz8dSh1GBtLNz7AqVd2v7tnVzo0iFPIlyqvhUiHQX7M0+lPOz0hQ581VlHw=
+stages:
+- lint
+- test
+- release
+cache:
+  directories:
+    - vendor
+    - rice-box.go
+jobs:
+  include:
+  - stage: lint
+    script: "./build/run_gometalinter.sh"
+  - stage: test
+    script: "./build/build_all.sh"
+    deploy:
+      provider: script
+      skip_cleanup: true
+      script:
+      - cp dockerfiles/filebrowser Dockerfile
+      - docker build -t filebrowser/filebrowser .
+      - echo "$DOCKER_PASS" | docker login -u "$DOCKER_USER" --password-stdin
+      - docker push filebrowser/filebrowser
+      - docker logout
+      on:
+        tags: false
+        repo: filebrowser/filebrowser
+        branch: master
+  - stage: release
+    script:
+      - cp dockerfiles/filebrowser Dockerfile
+      - echo "$DOCKER_PASS" | docker login -u "$DOCKER_USER" --password-stdin
+      - docker run --rm -itv $(pwd):$WDIR -v /var/run/docker.sock:/var/run/docker.sock filebrowser/dev goreleaser
+      - docker logout
+    if: tag IS present
+    deploy:
+      provider: releases
+      api_key:
+        secure: fEJu6vcxplTZ3ZR/fALLBp4mPWViIba4aeoiwNVJ9zqKmiZ5KG0pRqTMyevt8JUlR9yW0QNAfzkpd+wCqBHVWKIOy9Sy1Ocp3D0TY23ZAV29qKWwON9alBPqs6X6v6FiO0JFf5tARd+bpTmB3gFEUL1AGJ/UADv15fjwTERWlueloP6jmUCTCZEaTxkwrWxM8eBcMb3RlQykVKCMAZ5lGILWEKvJ8eBA7eP6GQtscCyS4tud+UWUb60VAufSPFv5K99dapm/1EXTTjgbtiY/zPP6FigR2RN/F6U0IUeEZbmftWcuDsp6lG4qInfwB1wK22F62KV193h8kgFDd8fB3hZuBVqBMtCxhuDFUFpULo2nAK8yyJP4/nc8SejHTxgFmIMhnUxLcE0om0E6bD6dFGNON08Bb/DheIvTzJSl44x6BomC/6xKKUp/GI6jGIxhV7YuWSJB4V3/5kG2ePbjEn/L+9BLY6L0kY7eea31xpSdDk881p0NOCN0lrD8RcDFf9qBtVhsh/RG2nbloC9mJI1iLTX/JRK4Rg1FROtsOE7PWPy3DbDB1rEweea2qahIbThbvfYbIhCYi0Oly6aeSJYfphQFY/hyNNrZZmWCq9gejVZelLDiYEV8Zkorhn4i/hPQ13nkVn1Rbd8jacoM1i3M4S4hkhjzlBIB+N2qudE=
+      file: "dist/*.tar.gz"
```

Comment:
It should be everything on dist, even the checksums and *.zip.

Reply:
Sure. I replaced it with `dist/*.*`. This will include `CHANGELOG.md` and `config.yaml` too. Is it ok?

``` bash
$ ls dist/*.*
dist/CHANGELOG.md                        dist/linux-armv5-filebrowser.tar.gz
dist/config.yaml                         dist/linux-armv6-filebrowser.tar.gz
dist/darwin-amd64-filebrowser.tar.gz     dist/linux-armv7-filebrowser.tar.gz
dist/dragonfly-amd64-filebrowser.tar.gz  dist/netbsd-386-filebrowser.tar.gz
dist/filebrowser_0.0.0_checksums.txt     dist/netbsd-amd64-filebrowser.tar.gz
dist/freebsd-386-filebrowser.tar.gz      dist/openbsd-386-filebrowser.tar.gz
dist/freebsd-amd64-filebrowser.tar.gz    dist/openbsd-amd64-filebrowser.tar.gz
dist/linux-386-filebrowser.tar.gz        dist/solaris-amd64-filebrowser.tar.gz
dist/linux-amd64-filebrowser.tar.gz      dist/windows-386-filebrowser.zip
dist/linux-arm64-filebrowser.tar.gz      dist/windows-amd64-filebrowser.zip
```

``` bash
$ tree dist
dist
├── CHANGELOG.md
├── config.yaml
├── darwin-amd64-filebrowser.tar.gz
├── dragonfly-amd64-filebrowser.tar.gz
├── filebrowser_0.0.0_checksums.txt
├── freebsd-386-filebrowser.tar.gz
├── freebsd-amd64-filebrowser.tar.gz
├── linux-386-filebrowser.tar.gz
├── linux-amd64-filebrowser.tar.gz
├── linux-arm64-filebrowser.tar.gz
├── linux-armv5-filebrowser.tar.gz
├── linux-armv6-filebrowser.tar.gz
├── linux-armv7-filebrowser.tar.gz
├── netbsd-386-filebrowser.tar.gz
├── netbsd-amd64-filebrowser.tar.gz
├── openbsd-386-filebrowser.tar.gz
├── openbsd-amd64-filebrowser.tar.gz
├── solaris-amd64-filebrowser.tar.gz
├── windows-386-filebrowser.zip
├── windows-amd64-filebrowser.zip
├── darwin_amd64
│   └── filebrowser
├── dragonfly_amd64
│   └── filebrowser
├── freebsd_386
│   └── filebrowser
├── freebsd_amd64
│   └── filebrowser
├── linux_386
│   └── filebrowser
├── linux_amd64
│   └── filebrowser
├── linux_arm_5
│   └── filebrowser
├── linux_arm_6
│   └── filebrowser
├── linux_arm64
│   └── filebrowser
├── linux_arm_7
│   └── filebrowser
├── netbsd_386
│   └── filebrowser
├── netbsd_amd64
│   └── filebrowser
├── openbsd_386
│   └── filebrowser
├── openbsd_amd64
│   └── filebrowser
├── solaris_amd64
│   └── filebrowser
├── windows_386
│   └── filebrowser.exe
└── windows_amd64
    └── filebrowser.exe
```